types/hooks/screens/useMessageHook.types.ts

```diff
@@ -2,9 +2,13 @@ import {ContextMenuOnPressNativeEvent} from 'react-native-context-menu-view';
 import {NativeSyntheticEvent} from 'react-native';
 
 interface UseMessageHook {
+  replyPreview: any;
+  setReplyPreview: (any) => void;
+  clearReplyPreview: () => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `replyPreview` and `setReplyPreview` properties are typed as `any`. It's recommended to avoid using `any` type as it can lead to potential runtime errors. Instead, define a specific type that describes the structure of the reply preview.

10:
The `data` parameter is also typed as `any`. If this represents a message, consider creating a Message type (if not already present) and use it here.

```diff
-  replyPreview: any;
-  setReplyPreview: (any) => void;
+  replyPreview: ReplyPreviewType; // Define ReplyPreviewType according to your data structure
+  setReplyPreview: (preview: ReplyPreviewType) => void;

-  data: any,
+  data: MessageType, // Define MessageType according to your data structure
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->